docs/ContribOperators.md

```diff
@@ -2912,7 +2912,7 @@ This version of the operator has been available since version 1 of the 'com.micr
 
 <dl>
 <dt><tt>T</tt> : tensor(uint8), tensor(int8)</dt>
-<dd>Constrain input and output types to singed/unsigned int8 tensors.</dd>
+<dd>Constrain input and output types to signed/unsigned int8 tensors.</dd>
```

Comment:
Update the source as well:
https://github.com/microsoft/onnxruntime/blob/6efa9d9e106c818020215988046aec86a829baf5/onnxruntime/core/graph/contrib_ops/quantization_defs.cc#L615